discovery-provider/src/models/users/audio_transactions_history.py

```diff
@@ -0,0 +1,38 @@
+import enum
+
+from sqlalchemy import BigInteger, Column, DateTime, Enum, String, text
+from src.models.base import Base
+from src.models.model_utils import RepresentableMixin
+
+
+class TransactionType(str, enum.Enum):
+    tip = "TIP"
+    reward = "REWARD"
+    transfer = "TRANSFER"
+    purchase_stripe = "PURCHASE_STRIPE"
+    purchase_coinbase = "PURCHASE_COINBASE"
+
+
+class TransactionMethod(str, enum.Enum):
+    send = "SEND"
+    receive = "RECEIVE"
+    user = "USER"
```

Comment:
I think these should be rolled into the `TransactionType` if we're doing the same for purchases? IE only send/receive for method?

Reply:
Not opposed but this is kind of the same old debate - at that point do we even need the method?
I'd almost rather add the method for purchases. Don't think it really matters but perhaps we should just pick one approach or the other and have all the transactions conform to that.

Reply:
My mental model was derived working backwards from the icons in the UI in Figma, which has different "little" icons for send/received/stripe/coinbase and the same "big" icons for tips/trending challenge/user challenge/purchase/transfer

but yea agree, we should just pick one and stick to it

Reply:
Actually.. re-reading the comment thread on the design doc, I'm sold on keeping the method just for send vs receive, and flatten out everything else.